vendor.conf

```
@@ -8,9 +8,9 @@ github.com/bugsnag/bugsnag-go b1d153021fcd90ca3f080db36bec96dc690fb274
 github.com/bugsnag/osext 0dd3f918b21bec95ace9dc86c7e70266cfc5c702
 github.com/bugsnag/panicwrap e2c28503fcd0675329da73bf48b33404db873782
 github.com/denverdino/aliyungo afedced274aa9a7fcdd47ac97018f0f8db4e5de2
-github.com/dgrijalva/jwt-go a601269ab70c205d26370c16f7c81e9017c14e04
```

I see updating Azure/go-autorest brings a substantial number of code changes (for a patch release).

If the bug is in this library, could we instead update just this library to a version with the fix? I see maintains a fork with the fix (IIUC), and we can specify it with a custom location (the equivalent to `replace` in `go.mod`);

```suggestion
github.com/dgrijalva/jwt-go a211650c6ae1cff6d7347d3e24070d65dcfb1122 https://github.com/form3tech-oss/jwt-go.git # v3.2.4
```

That would only bring the diff of the jwt-go package;
https://github.com/form3tech-oss/jwt-go/compare/a601269ab70c205d26370c16f7c81e9017c14e04...v3.2.4

---

gave it a quick attempt; https://github.com/distribution/distribution/pull/3466

---

Not against using a replace if vndr can do that. Should we consider using the fork which the original repository now links to? https://github.com/golang-jwt/jwt

---

Oh! Arf... there's _two_ forks now, and both being actively maintained? 😞

---

I updated https://github.com/distribution/distribution/pull/3466 with a commit (allowing the differences between those forks to be reviewed)